README.md

```diff
@@ -1,6 +1,9 @@
 # Blockchain Demo
 A web-based demonstration of blockchain concepts.
 
+_NOTE: If you want to read a documentation in Brazilian Portuguese [click this link](https://blockchain4devs.github.io/blockchain-demo/docs/pt-br/).
```

Comment:
Can we do this in another way? Ideally there is a markdown version of the docs that we look to locally on GitHub. Thoughts?